.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
```

Comment:
We use prettier, please setup prettier on ur editor and this wont be needed, will add this to contribution guidlines to make it clearer for future devs.

Reply:
Sounds good. Loosely related, there *might* be an issue in which prettier and eslint are battling each other work during the pre commit hook. I've seen some strange behavior there, but I'm not certain about it.